CONTRIBUTING.rst

```diff
@@ -95,7 +95,7 @@ small size, and usually don't require a broad knowledge of the edX platform.
 It makes them good candidates for a first task, allowing you to focus on getting
 familiar with the development environment and the contribution process.
 
-.. _byte-sized bugs and tasks: http://bit.ly/edxbugs
+.. _byte-sized bugs and tasks: https://github.com/search?q=user%3Aopenedx+label%3A%22help+wanted%22&type=Issues&ref=advsearch&l=&l=
```

Comment:
@sarina It could be worth picking one of the two? I'm fine with either, but having a clear standard choice would be good imho :) CC @regisb

Reply:
I think it's easier to say, these two labels work, rather than somehow prohibiting the whole org from using a tag.

Reply:
Perhaps we could use two for now, and revisit this documentation if/when we successfully standardize?

Reply:
@sarina What is preventing us from doing the standardization now? It's not about prohibiting, but rather having a clear choice that we can use going forward, rather than keeping several different tags? Happy to help with it, it would be good to clean this up fully while we are at it

Reply:
My thought is people will still use the label they're used to. I don't know how to solve that problem, and I don't have the bandwidth currently to do so. If you're convinced you can do the work to clean things up and enforce it, great.